index.html

```diff
@@ -215,21 +215,55 @@ <h3 class="section-title contact-title">CONTACT</h3>
         </p>
       </div>
       <div class="contact-form">
-        <form action="#">
-          <input type="text" name="text" id="text" placeholder="Text Input" />
+        <form action="https://formspree.io/f/xayvleal" method="post">
+          <input
+            type="text"
+            name="user_name"
+            id="user_name"
+            maxlength="30"
+            required
+            placeholder="Text Input"
+          />
           <input
             type="email"
-            name="email"
-            id="email"
+            required
+            name="user_email"
+            id="user_email"
             placeholder="jeremy@gmail.com"
             value="jeremy@gmail.com"
```

Comment:
- [ ] Here too it's better to put a descriptive placeholder such as `Enter your Email`. Also, I think adding a value to the email field is not a very good idea because the email address is different for each and every user so its better to just put the placeholder